group_vars/location_beuth/networks.yml

```diff
@@ -0,0 +1,94 @@
+---
+ipv6_prefix: "2001:bf7:750:1200::/56"
+
+networks:
+  - vid: 111
+    role: mesh
+    name: mesh_segen
+    prefix: 10.230.23.141/32
+    ipv6_subprefix: -1
+
+  - vid: 112
+    role: mesh
+    name: mesh_perleberger36
+    prefix: 10.230.23.142/32
+    ipv6_subprefix: -2
+
+  - vid: 113
+    role: mesh
+    name: mesh_scherer8
+    prefix: 10.230.23.143/32
+    ipv6_subprefix: -3
+
+  - vid: 114
+    role: mesh
+    name: mesh_nord
+    prefix: 10.230.23.144/32
+    ipv6_subprefix: -4
+
+  - vid: 115
+    role: mesh
+    name: mesh_nord
+    prefix: 10.230.23.145/32
+    ipv6_subprefix: -5
+
+  - vid: 116
+    role: mesh
+    name: mesh_jup
+    prefix: 10.230.23.146/32
+    ipv6_subprefix: -6
+
+  - vid: 117
+    role: mesh
+    name: mesh_ost
+    prefix: 10.230.23.147/32
+    ipv6_subprefix: -7
+
+  - vid: 118
+    role: mesh
+    name: mesh_sued
+    prefix: 10.230.23.148/32
+    ipv6_subprefix: -8
+
+  - vid: 119
+    role: mesh
+    name: mesh_fardf
+    prefix: 10.230.23.149/32
+    ipv6_subprefix: -9
+
+  - vid: 128
+    role: mesh
+    name: mesh_west
+    prefix: 10.230.23.158/32
+    ipv6_subprefix: -28
+
+  - vid: 104
+    role: dhcp
+    prefix: 10.230.23.152/29
+    ipv6_subprefix: 0
+    untagged: true
+    inbound_filtering: true
```

Comment:
I don't think that we need inbound filtering on beuth. We agreed on doing that in house projects, but not on public sites.